src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -77,6 +84,11 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
         send("TEXT_UPDATED", { data: { value } });
     }, [debouncedValue]);
 
+    const onValueChanged = (value: Descendant[]): void => {
+        setValue(value);
+        setWrapperStyle(getMinWidthIfEmpty(editor, placeholder, wrapperRef.current));
```

Comment:
could you add a check if there is placelholder to only handle it in this case?

Reply:
Added the check inside `getMinWidthIfEmpty`